src/app/client/src/app/modules/player-helper/components/content-rating/content-rating.component.html

```diff
@@ -32,9 +32,9 @@
             <input type="checkbox" role="checkbox" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
             <label [class.options-content-checked]="feedback.checked">{{feedback['value']}}</label>
           </div> -->
-          <div class="sb-checkbox sb-checkbox-primary sb-checkbox-rightaligned" *ngFor="let feedback of feedbackObj[contentRating]['options']">
-            <input type="checkbox" id="{{feedback['value']}}" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
-            <label for="{{feedback['value']}}" class="fsmall mb-16">{{feedback['value']}}</label>
+          <div class="sb-checkbox sb-checkbox-primary sb-checkbox-rightaligned" *ngFor="let feedback of feedbackObj[contentRating]['options'];index as i">
+            <input type="checkbox" attr.aria-labelledby="feedbackRating{{i}}" id="feedbackRating{{i}}" [(ngModel)]="feedback.checked" (click)="changeOptions(feedback)">
```

Comment:
what is i and value why is it diffenrent?

Reply:
![Screenshot (113)](https://user-images.githubusercontent.com/29474792/110771890-08810f80-8281-11eb-84b4-2b7b3e78fb42.png)

The html id come with string and space,  That is invalid id 's value must not contain whitespace
@rajeevsathish